ship_traffic/ship_traffic.ipynb

```diff
@@ -341,7 +343,7 @@
     "points = hd.dynspread(hd.datashade(pts, aggregator=ds.count_cat('category'), color_key=color_key))\n",
     "\n",
     "tiles  = hv.element.tiles.ESRI().opts(alpha=0.4, bgcolor=\"black\").opts(responsive=True, min_height=600)\n",
-    "labels = hv.element.tiles.StamenLabels().opts(alpha=0.7, level='glyph')\n",
+    "labels = hv.element.tiles.EsriUSATopo().opts(alpha=0.4, level='glyph')\n",
```

Comment:
@Azaya89 the code used to overlay an ESRI base map with the Stamen Labels layer that just contains labels (https://maps.stamen.com/toner-labels/#12/37.7922/-122.2301). With these changes, it is now overlaying two ESRI base maps, which doesn't make much sense. We've also lost the labels.

Let's go through these examples next week together.

Reply:
OK, I was thinking about it in a different way. See you next week then!

Reply:
It would be great if we could find a labels-only tile set that still works, but I'm not aware of any. Failing that, I think the task is to choose an underlying tile set with labels embedded in it already in a way that is not too distracting. I do think people need labels, because otherwise it's hard to orient yourself. But with many tile sets, the labels are rendered in a way that becomes extremely busy when dense data is overlaid on them. So there's definitely some trial and error involved and tradeoffs to make.

Reply:
I might have found a label layer that fits this purpose, it's `CartoDB.PositronNoLabels` made available by `xyzservices` (https://xyzservices.readthedocs.io/en/stable/gallery.html).

Screenshot 1:
![image](https://github.com/user-attachments/assets/8496c3d7-532d-4978-8781-8aa099863e3f)
Screenshot 2:
![image](https://github.com/user-attachments/assets/9bd816dd-a5c7-4a1c-9430-d9c17f3d0675)

Reply:
Oops yes sorry, I pasted the wrong name but meant `CartoDB.PositronOnlyLabels` as shown in the screenshots.

Reply:
Nice! The CartoDB.VoyagerOnlyLabels from the same site also look good, and CartoDB.DarkMatterOnlyLabels could be good for predominately light plots. I wonder if they have limits on how many served per day or hour or whatever, because some of the tiles I tried to load worked, but others later did not.

<img width="764" alt="image" src="https://github.com/user-attachments/assets/a3e0b9ba-da0c-4a4c-885a-6296d9d46f1d">

<img width="776" alt="image" src="https://github.com/user-attachments/assets/e899fbc0-443e-4438-aefb-191d63b92570">